src/views/AppRoot/component.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { BrowserRouter, Route, Redirect } from 'react-router-dom'
+
+import LoginScreen from '../pages/Login'
+import Dashboard from '../pages/Dashboard'
+import StubsRoot from '../components/stubs/StubsRoot'
+
+const AppRootComponent = ({ isLoggedIn }) => (
+  <BrowserRouter>
+    <Route
+      path="/login"
+      component={LoginScreen}
+    />
+    <Route
+      path="/"
+      component={Dashboard}
+    />
+    <Route
+      path="/stubs"
+      component={Dashboard}
+    />
+    {!isLoggedIn && <Redirect to="/login" />}
```

Comment:
let's create `PrivateRoute` component, which will handle redirect of non logged in users
and `GuestRoute` component, which will handle redirect of logged in users from auth pages

Reply:
Ok, I work on it, thanks